docs/preview/02-Features/writing-different-telemetry-types.md

```diff
@@ -388,6 +388,19 @@ catch (Exception exception)
 }
 ```
 
+### Making it easier to link dependencies
+
+Service-to-service correlation requires linkage between tracked dependencies (outgoing) and requests (incoming).
+Tracking any kind of dependency with the library has the possibility to provide an dependency ID. This ID will be needed later when the incoming request is tracked (dependency ID = request's parent ID).
+
+// TODO: provide example.
```

Comment:
Think the example still needs to be done :)

Reply:
Ok, I updated this, but we also should probably create a dedicated feature doc page to give a rundown of the entire service-to-service functionality. But that's for the future 😉 . I'll create an issue for this.

Reply:
#258